refactor(main): separate some reused logic

pan93412 opened the pull request:
tag: `test needed`

pan93412 committed:
refactor(main): separate some reused logic

rogeraabbccdd commented on js/app.js:
```diff
@@ -75,11 +74,11 @@ const app = Vue.createApp({
+        (difficulty.value === 1 && valueAbsRange(value, 60) ||
         // 超級簡單
-        (difficulty.value === 2 && (value > 1080 || value < -1080))
+        (difficulty.value === 2 && valueAbsRange(value, 1080))
```
多一個 `)`

pan93412 replied:
```javascript
(
  difficulty.value === 2
  && valueAbsRange(value, 1080)
)
```

沒有吧？ 🤔

rogeraabbccdd replied:
🤔 
`Uncaught SyntaxError: Unexpected token '{'`  `app.js:82 `

rogeraabbccdd reviewed (commented):
有個小問題

pan93412 committed:
fix(app): SyntaxError: Unexpected token '{'

a possible fix to https://github.com/rogeraabbccdd/AsiaGodTone-Hotpot-Game/pull/10#discussion_r584238642

rogeraabbccdd commented on js/app.js:
```diff
@@ -26,8 +28,13 @@ const app = Vue.createApp({
+      audio.currentTime = 0
+      audio.src = src
+      audio.loop = repeat
+      audio.play()
```
![chrome_PVEkvkGSVC](https://user-images.githubusercontent.com/15815422/109445411-7efc6100-7a7a-11eb-995d-2ea9817e526c.png)

rogeraabbccdd reviewed (commented):
有錯誤?
函式參數與呼叫時傳入的東西不符
```js
const playAudio = (audio, src, repeat = false)
```
```js
playAudio('./assets/again.mp3')
```

pan93412 commented:
> 有錯誤?

好像是 playAudio 那邊忘記傳入 audio。等我改一下

pan93412 commented:
> 有錯誤?
> 函式參數與呼叫時傳入的東西不符
> ```js
> const playAudio = (audio, src, repeat = false)
> ```
> ```js
> playAudio('./assets/again.mp3')
> ```

你可以幫我把 playAudio 那邊的 audio 參數刪掉嗎？因為上層有定義過 audio 所以是多餘的。

pan93412 committed:
fix(app): the meaningless "audio" parameter

pan93412 committed:
Merge branch 'master' into patch-2
